src/app/_common-components/menu/menu.component.ts

```diff
@@ -121,11 +123,20 @@ export class MenuComponent implements AfterViewInit, OnDestroy {
 
     this.resizeObserver = new ResizeObserver((entries) => {
       this.width$.next(Math.floor(entries[0].contentRect.width));
+      setTimeout(() => {
+        this.fixResizeInkBar();
+      }, 500);
```

Comment:
is it not working with 0 setTimeout? 500 seems a little arbitrary and maybe will not work on all clients

Reply:
We currently need it for the animation. It won't work with 0. Tested it on multiple browser clients and it's working.